README.md

```diff
@@ -1,2 +1,57 @@
-# web-development-course
-This project is aimed to create an online web development courses platform.
+Online Web development courses
+
+The project is inspired by the FrontEdMasters website (https://frontendmasters.com/)
+For this online Web developmen courses website, I implemented Html,SCSS And javaScript that includes the project details in both mobile and desktop.
+
+ 
+## Built With the following
+
+- HTML
+- SaSS
+- javaScript
+- Linters
+
+## Get Started
+
+Please get your browser update.
+To get a local copy up and running follow these simple steps.
+
+Run the command below from the Terminal:
+
+      git clone <URL> 'https://github.com/degisew/web-development-course' in your terminal
+
+	  cd platform-feature
+      
+
+## Live Demo
+
```

Comment:
Great job @degisew 🤝

- [ ] [Optional] Kindly remove this live demo link. It doesn't appear to be specifically for this project.